package.json

```
@@ -8,5 +8,8 @@
   },
   "keywords": [],
   "author": "",
-  "license": "ISC"
+  "license": "ISC",
+  "dependencies": {
+    "spago": "^0.20.9"
+  }
```

This change is not needed

```suggestion
  "license": "ISC"
```